### Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not



In [2]:
import numpy as np
from scipy import stats
import scipy.stats as st

In [6]:
# lets perform a Z-test
# 𝐻0: 𝜇 = 100, 𝐻1: 𝜇  >   52
mean = 100
assumed_mean = 108
n = 36
std_dev = 15
alpha = 0.05
std_error = std_dev/np.sqrt(n)
z_score = (assumed_mean-mean)/ std_error
print("Z_score : ",z_score)




Z_score :  3.2


In [4]:
area_above = 1-st.norm.cdf(z_score)
print("area above z-score {} is {}".format(z_score,area_above))

area above z-score 3.2 is 0.0006871379379158604


In [5]:
alpha=0.05
if(area_above>alpha):
    print("Fail to reject H0")
else:
    print("Reject H0")

Reject H0


### Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple random sample of 100 voters are surveyed from each state.

What is the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state?



In [7]:
# H0: t
# H1: there is difference between duracell and Energizer
n1 = 100
p1 = 0.52
p2 = 0.47
n2 = 100

SE = np.sqrt(( p1*(1 - p1) / n1 ) + ( p2*(1 - p2) / n2 ))

z_score = -(p1-p2) / SE
print("Z_Score is", z_score)

print("probabiliy of z-score {} is {}".format(z_score,st.norm.cdf(z_score)))



Z_Score is -0.7080278164104213
probabiliy of z-score -0.7080278164104213 is 0.23946399182220013


### Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard deviation is 209. How well did you score on the test compared to the average test taker?



In [8]:
x=1100
std_dev = 209
mean = 1026

z_score = (x-mean)/std_dev
print("Z_Score is", z_score)

print("probabiliy of z-score {} is {}".format(z_score,st.norm.cdf(z_score)))



Z_Score is 0.35406698564593303
probabiliy of z-score 0.35406698564593303 is 0.6383556584353189


we can say that score is 0.35 standard deviations higher than average test taker